examples/asr/conf/asr_adapters/asr_adaptation.yaml

```diff
@@ -82,7 +84,9 @@ model:
     # This can be useful in order to select *which type of adapter* is added, *what adapters to enable*,
     # and further global operations that can decide dynamically how to support the requested adapter.
     global_cfg:
-      encoder_adapter: True  # ASR adapter key, determines whether to add encoder adapter modules
+      check_encoder_adapter: False  # ASR adapter key, determines whether to check if encoder adapter modules is supported
```

Comment:
These flags will now only check for the value rather than actually determine if the adapter is added or not. The issue becomes if they test a ctc model for joint adapters (not possible), then the adapter script will crash. During restoration, if adapters are not supported then restore will fail even if all adapters are disabled. So it is tricky.

I can do one thing, which is instead of crashing or raising error, I can do a one time log stating that while checking, the module was found to not support adapters.

Reply:
Done